src/main/blackbox/blackbox.c

```diff
@@ -1090,6 +1090,7 @@ static bool blackboxWriteSysinfo(void) {
     switch (xmitState.headerIndex) {
         BLACKBOX_PRINT_HEADER_LINE("Firmware type", "%s",                   "Cleanflight");
         BLACKBOX_PRINT_HEADER_LINE("Firmware revision", "%s %s (%s) %s",    FC_FIRMWARE_NAME, "3.7.0", shortGitRevision, targetName);
```

Comment:
will this "tricking" of BBL ever end? are we to the point that we can display the correct firmware version and use the emu bbl instead?

Reply:
What do you mean nu tricking? I made already that we can show correct version :) I need target to separate IMuF for helio and non helio in headers

Reply:
"3.7.0" we are telling it that we are using version 0.3.7 which is clearly shown here 
![image](https://user-images.githubusercontent.com/46289813/124339410-999b2f80-db6b-11eb-90a9-32709baf3e8c.png)

Reply:
https://github.com/emuflight/EmuFlight/pull/624
This need to be merged. @nerdCopter want to wait before EBB completed completely